server/forms.py

```diff
@@ -81,6 +81,10 @@ class SelectAssetForm(forms.Form):
                 "stellar:SRT:GCDNJUBQSX7AJWLJACMJ7I4BC3Z47BQUTMHEICZLE6MU4KQBRYG5JY6B",
                 "Stellar Reference Token (SRT)",
             ),
+            (
+                "stellar:USDC:GBD6ZRRI4FZW55G7NBRMF2NY2DANKHXY63J4KKIQT5GWNWEUZWZ6MS2A",
```

Comment:
This is not Circle's testnet address, can we use the real testnet USDC?

Reply:
`GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5`

Reply:
We obviously wouldn't be able to issue USDC, but we can hold a balance and get USDC from their faucet.